src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -284,6 +317,63 @@ private String md5Hash(final byte[] bytes) {
         );
     }
 
+    /**
+     * Load the file ALT_NAMES containing the alternative contig names
+     * @author Pierre Lindenbaum
+     * @return a <code>Map&lt;src_contig,Set&lt;new_names&gt;&gt;</code>. Never null. May be empty if ALT_NAMES is null.
+     * @throws PicardException if there is any error in the file ALT_NAMES
+     */
+    private Map<String, Set<String>> loadContigAliasesMap() throws PicardException {
+    	// return an empty map if no mapping file was provided
+    	if (this.ALT_NAMES == null) return Collections.emptyMap();
```

Comment:
Please make the formatting changes from my first set of comments:
- `if` statements should be multi-line and have `{` `}` around the body
- `for(` should be written as `for (`
- `catch` should be wrapped up to the line before it, as `} catch (...) {`
- `//` comments should have a space between the `//` and the first letter of the comment